export.py

```diff
@@ -64,43 +64,44 @@ def main(argv=None):
         print >> sys.stderr, sys.argv[0].split("/")[-1] + ": " + str(err)
         print >> sys.stderr, "\t for help use --help"
         return 2
-    
+
     if login is None:
         raise Exception('Login parameter must be present.')
-    
+
     if password is None:
         raise Exception('Password parameter must be present.')
-    
+
     bitly = Bitly(login, password, verbose)
-    
+
     limit = 100
     offset = 0
     result_count = 0
     nb_found = 0
-    
+
     csv_writer = csv.writer(open(output_path, 'wb'), quoting=csv.QUOTE_ALL)
     csv_writer.writerow(('Link', 'Long url'))
```

Comment:
Given the change to the `CSV` structure below, you should update the header row here.